src/main/java/uk/gov/companieshouse/model/Appeal.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.companieshouse.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+@Document(collection = "appeals")
+public class Appeal {
+
+    @JsonIgnore
+    @Id
+    private String id;
+
+    @JsonIgnore
+    private LocalDateTime createdAt;
+
+    @JsonIgnore
+    private CreatedBy createdBy;
+
+    @Valid
+    @NotNull(message = "penaltyIdentifier must not be null")
+    private PenaltyIdentifier penaltyIdentifier;
+
+    @Valid
+    @NotNull(message = "reasons must not be null")
+    @JsonProperty("reasons")
+    private Reason reason;
+
+    public Appeal() {
```

Comment:
Please remove default constructor as it is always there by default.

Reply:
Priority: should change.

Reply:
i just quickly run 'de-lombok', will tidy up

Reply:
Nasty tools :P

Reply:
done